style-sheets/carbon-components-overrides.scss

```diff
@@ -10,6 +14,14 @@
   $button-border-radius: 0.25rem
 );
 
+@use '@carbon/styles/scss/grid';
+
+@include grid.flex-grid();
+
+body {
+  font-family: map.get(typeFamily.$type-family, 'sans');
```

Comment:
oh Maria @mamartinezmejia, you want to test this locally first, previous we're getting some errors, refers this issue: https://github.com/bcgov/nr-fsa-theme/issues/34. make sure it works fine locally

Reply:
Oh, yes, we did it. Thanks for checking.